### ISSUE TITLE
On GoPro video upload by user, extract GPMF track and save as txt file in sequence directory

### ISSUE THREAD

**Issue by himynamesdave** [trek-view/mtp-desktop-uploader#28]
As pe

https://github.com/JuanIrache/gpmf-extract

**Comment by superdev0505** [comment]
Just enable https://github.com/JuanIrache/gpmf-extract to our app is okay?

**Comment by superdev0505** [comment]
https://github.com/gopro/gpmf-parser is not based on python or javascript.
We have this part as python script?

**Comment by himynamesdave** [comment]
> Just enable https://github.com/JuanIrache/gpmf-extract to our app is okay?

Yes. If metadata type is gmpf, then perfrom this action, and output as gpmf in output dir.

ONLY apply to video files where 

```
Meta Format                     : gpmd
```

See this post: https://www.trekview.org/blog/2020/metadata-exif-xmp-360-video-files/
